packages/core/src/rum/instrumentation/DdRumResourceTracking.tsx

```diff
@@ -234,6 +245,31 @@ export class DdRumResourceTracking {
         }
     }
 
+    static getTracingAttributes(url: string): DdRumResourceTracingAttributes {
+        try {
+            const hostname = new URL(url).hostname;
+
+            if (DdRumResourceTracking.firstPartyHosts.includes(hostname)) {
```

Comment:
Hmm...I am afraid that this check is not going to handle also the subdomains:
`domain.com` and `subdomain.domain.com`

Reply:
Yes, I figured this with Nikita, I have made the change, using the iOS SDK regex